### DISCUSSION TITLE
xmake install -o /targetdir 子目录问题

### DISCUSSION THREAD

**Discussion by DanielKui** [xmake-io/xmake#4011] [Q&A] [answered]
![image](https://github.com/xmake-io/xmake/assets/578118/1372baff-5da8-481b-bcea-2380d1600d73)

我添加了add_packages()，xmake install -o ~/testbin后，拷贝的.so 进行了分类bin  lib，是否可以不需要分类，混在一起呢？

**Comment by DanielKui** [comment]
比如直接拷贝到~/testbin/下即可，不用细分，这个如何设置呢？

**Reply by waruqi** [reply, marked as answer]
自己 on_install 重写安装逻辑，自己去 copy
